ansible/README.md

```diff
@@ -25,7 +25,7 @@ brew install python
 pip install ansible==2.0.2.0
 
 cd ansible
-ansible-playbook setup.yml [-e docker_machine_name=whisk]
+ansible-playbook -i environments/mac setup.yml [-e docker_machine_name=whisk]
```

Comment:
Ah I think what's going on, and someone else hit the same problem in my team. 
I think part of the problem is the naming of the steps we "prereq" and "setup"  and it's hard to remember which what order they need to run. 
I will continue to use it as is but might come up with a simpler naming and ordering. Right people need to pay attention to the order and run setup and then prereq

I'm not questioning on what each do, just giving feedback that folks are getting both names mixed